subscribe.go

```diff
@@ -135,7 +137,11 @@ func (h *Hub) registerSubscriber(w http.ResponseWriter, r *http.Request) *Subscr
 	sendHeaders(w, s)
 
 	if c := h.logger.Check(zap.InfoLevel, "New subscriber"); c != nil {
-		c.Write(zap.Object("subscriber", s))
+		if h.debug {
```

Comment:
IMHO you should check the debug level of the logger instead of relying on `h.debug`. This will allow user wanting to log detailed information in "prod" mode to do it.

You can use https://pkg.go.dev/go.uber.org/zap#Logger.Level to do the check.